cristin-biobank/src/main/java/no/unit/nva/biobank/model/nva/ExternalSourcesBiobank.java

```diff
@@ -0,0 +1,35 @@
+package no.unit.nva.biobank.model.nva;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import no.unit.nva.cristin.model.CristinExternalSource;
+
+public class ExternalSourcesBiobank {
+
+
+    public static final String EXTERNAL_SOURCE_SHORT_NAME = "sourceShortName";
+    public static final String EXTERNAL_SOURCE_REDERENCE_ID = "sourceReferenceId";
```

Comment:
Ask other reviewer if we should use these names for the data model

Reply:
I would have called this class `ExternalSource`. And the fields `source` and `reference`. Is it a similar model elsewhere in the Cristin API?